airbyte-integrations/connectors/source-amazon-ads/acceptance-test-config.yml

```diff
@@ -38,3 +38,5 @@ tests:
       ignored_fields:
         "sponsored_products_report_stream": ["updatedAt"]
       timeout_seconds: 3600
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog_attribution_report.json"
```

Comment:
so, `secrets/config_report.json` looks like a secret for report, a little different than the one for `integration_tests/configured_catalog.json`, can you merge the configured catalog to either `configured_catalog.json` or `configured_catalog_report.json`

Reply:
Can you please differentiate between contents of files `secrets/config_report.json` and `secrets/config.json`.

Merging them with existing catalogue would still use the same secret file. 

I would like to keep it segregated for attribution reports unless they are causing any issues with integration test suite.

Reply:
`configured_catalog.json` uses `secrets/config.json`, and `configured_catalog_report.json` uses `secrets/config_report.json`, if `configured_catalog_attribution_report.json` also uses `secrets/config.json`, can you merge the json to `integration_tests/configured_catalog.json`?

Reply:
have merged it, but if you can share the difference between contents of files `secrets/config_report.json` and `secrets/config.json`, it will be helpful to understand. Thanks

Reply:
The CI may use multiple secret sets, but I only got one set of secrets. I will ask IT to share it with me. In my local environment, I use the same content for the both. 

You mentioned that you were stuck in 70%, I am able to reach 73% after fixing the `test_backward_compatibility`, here's my change:  https://github.com/YiyangLi/airbyte/commit/9341fc62a69b90ac114568b5d66a2faefba9d9bd  since we are using different accounts, the fix doesn't help you too much. Can you try again by removing `expect_records`? 

I am currently stuck at 73%, `TestBasicRead.test_read`, even if I remove `basic_read` section, I will be stuck at `incremental read`. I wonder if that's different on your side, and I may fix the dev account on our side.

Reply:
After pulling the latest from master it's going till 72%, before getting stuck at `TestBasicRead.test_read[inputs0]`

I tried with removing `expect_records` section, but same results.

When I removed `basic_read` section, success % reached till 82% before getting stuck at `incremental read`.